boards/arm/nrf9160dk_nrf9160/doc/index.rst

```diff
@@ -1,16 +1,15 @@
 .. _nrf9160dk_nrf9160:
 
-nRF9160 DK
-##########
+nRF9160 Development Kit (DK)
```

Comment:
Should we do same changes in documentation of nRF52840 DK and nRF51 DK?

Reply:
We should, yes. Maybe as part of renaming the remaining nrf52 boards?